product_security_main/hooks.py

```diff
@@ -0,0 +1,27 @@
+# # © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# # License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo.api import SUPERUSER_ID, Environment
+
+
+def uninstall_hook(cr, registry):
+    env = Environment(cr, SUPERUSER_ID, {})
+    access_ids = [
+        "product.access_product_template_manager",
+        "product.access_product_product_manager",
+        "stock.access_product_template_stock_manager",
+        "stock.access_product_product_stock_manager",
+        "mrp.access_product_template_mrp_manager",
+        "mrp.access_product_product_mrp_manager",
+        "sale.access_product_template_sale_manager",
+        "sale.access_product_product_sale_manager",
+        "purchase.access_product_template_purchase_manager",
+        "purchase.access_product_product_purchase_manager",
+        "account.access_product_template_account_manager",
+        "account.access_product_product_account_manager",
+    ]
+
+    for xml_id in access_ids:
+        record = env.ref(xml_id, raise_if_not_found=False)
+        if record:
+            record.write({"perm_create": True})
```

Comment:
Better use ORM write multi method capabilities (not tested, could you try?):
```suggestion
    rules = self.env["ir.rule"].browse([])
    for xml_id in access_ids:
        rules |= env.ref(xml_id, raise_if_not_found=False)
    if rules:
        rules.write({"perm_create": True})
```

Reply:
Done. but the change targets `ir.model.access` instead of `ir.rule`, as this was not the actual source of the issue.